src/Chapter1.hs

```diff
@@ -620,8 +633,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let restAndLast    = divMod (abs n) 10
+        restAndPreLast = divMod (fst restAndLast) 10
```

Comment:
As you only need the `mod` part of `divMod`, you can just use `mod` function on its own in here:

```haskell
        preLast = mod (fst restAndLast) 10
```